x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -1,80 +1,146 @@
-password authentication for {kib}.
-
-// tag::minimal-security-note[]
-IMPORTANT: The minimal security scenario is not sufficient for
-<<dev-vs-prod-mode,production mode>> clusters. If your cluster has multiple
-nodes, you must enable minimal security and then
-<<security-basic-setup,configure Transport Layer Security (TLS)>> between
-nodes.
+=== Prerequisites
 
-// end::minimal-security-note[]
-
-<<security-minimal-setup,Set up minimal security>>
+Download the `elasticsearch 8.0.0-alpha1` package distribution for your
+environment.
 
 [discrete]
-[[security-basic-overview]]
-=== Basic security ({es} Production)
-
-This scenario builds on the minimal security requirements by adding transport
-Layer Security (TLS) for communication between nodes. This additional layer
-requires that nodes verify security certificates, which prevents unauthorized
-nodes from joining your {es} cluster.
-
-Your external HTTP traffic between {es} and {kib} won't be encrypted, but
-internode communication will be secured.
-
-<<security-basic-setup,Set up basic security>>
+=== Start {es} with security enabled
+
+. Before starting {es} for the first time, run the
+`elasticsearch-security-config` tool from the directory where you downloaded
+the {es} package distribution:
++
+[source,shell]
+----
+./bin/elasticsearch-security-config
+----
+
+. Navigate to `$ES_PATH_CONF/auto_config_on_<timestamp>` to view the
+certificates and keys that the `elasticsearch-security-config` tool generated:
++
+--
+`http.p12`::
+Certificate for the HTTP layer, which you can use to enroll additional clients
+such as {kib}.
+
+`http_keystore_local_node.p12`::
+Keystore for the first {es} node in your cluster.
+
+`transport_keystore_all_nodes.p12`::
+Keystore for adding additional nodes to your cluster.
+--
++
+NOTE: The `ES_PATH_CONF` variable is the path for the {es}
+configuration files. If you installed {es} using archive distributions
+(`zip` or `tar.gz`), the variable defaults to `ES_HOME/config`. If you used
+package distributions (Debian or RPM), the variable defaults to
+`/etc/elasticsearch`.
+
+. Start {es}.
++
+[source,shell]
+----
+./bin/elasticsearch
+----
+
+. In another terminal window, run the
+<<setup-passwords,`elasticsearch-setup-passwords`>> utility from the directory
+where you downloaded the {es} package distribution:
++
+[source,shell]
+----
+./bin/elasticsearch-setup-passwords auto
+----
++
+If you want to use your own passwords, run the command with the
+`interactive` parameter instead of the `auto` parameter. Using this mode
+steps you through password configuration for all of the built-in users.
++
+[source,shell]
+----
+./bin/elasticsearch-setup-passwords interactive
+----
+
+. Save the generated passwords. You'll need them to add the built-in user to
+{kib}.
++
+WARNING: After you set a password for the `elastic` user, you cannot run the
+`elasticsearch-setup-passwords` command a second time.
```

Comment:
Add a note that if they want to change the password of the elastic user or if they lost it or they didn't write it down, they can always use `bin/elastisearch-reset-elastic-password`

Reply:
I'll change this WARNING to a NOTE, and then indicate how to change the password for the `elastic` user with the `elastisearch-reset-elastic-password` tool.